tests/vectors/main.rs

```diff
@@ -117,7 +145,7 @@ mod bip44_tests {
 
     #[derive(Debug, Deserialize)]
     struct Vector {
-        path: HDPath,
+        path: BIP44LikePath,
```

Comment:
same here as above...  need higher level paths, which then needed to be more lenient (canonical OK now)